akka-docs/src/main/paradox/scala/guide/introduction.md

```diff
@@ -1,49 +1,44 @@
+
+Akka's use of the actor model provides a level of abstraction that makes it
+easier to write correct concurrent, parallel and distributed systems. The actor
+model spans the full set of Akka libraries, providing you with a consistent way
+of understanding and using them. Thus, Akka offers a depth of integration that
+you cannot achieve by picking libraries to solve individual problems and trying
+to piece them together.
+
+By learning Akka and how to use the actor model, you will gain access to a vast
+and deep set of tools that solve difficult distributed/parallel systems problems
+in a uniform programming model where everything fits together tightly and
 efficiently.
 
-## What is the Actor Model?
+## How to get started
 
-The characteristics of today's computing environments are vastly different from the ones in use when the programming
-models of yesterday were conceived. Actors were invented decades ago by @extref[Carl Hewitt](wikipedia:Carl_Hewitt#Actor_model).
-But relatively recently, their applicability to the challenges of modern computing systems has been recognized and
-proved to be effective.
+If this is your first experience with Akka, we recommend that you start by
+running a simple Hello World project. See the *[QuickStart Guide
+Scala](http://developer.lightbend.com/guides/akka-quickstart-scala)* or
+*[QuickStart Guide
+Java](http://developer.lightbend.com/guides/akka-quickstart-java)* for
+instructions on downloading and running the Hello World example. The *QuickStart* guide walks you through example code that introduces how to define actor systems, actors, and messages as well as how to use the test module and logging. Within 30 minutes, you should be able to run the Hello World example and learn how it is constructed.
 
-The actor model provides an abstraction that allows you to think about your code in terms of communication, not unlike
-people in a large organization. The basic characteristic of actors is that they model the world as stateful entities
-communicating with each other by explicit message passing.
-
-As computational entities, actors have these characteristics:
-
-* They communicate with asynchronous messaging instead of method calls
-* They manage their own state
-* When responding to a message, they can:
-    * Create other (child) actors
-    * Send messages to other actors
-    * Stop (child) actors or themselves
+This *Getting Started* guide provides the next level of information. It covers why the actor model fits the needs of modern distributed systems and includes a tutorial that will help further your knowledge of Akka. Topics include:
+* [[Why modern systems need a new programming model|actors-motivation]]
+* [[How the actor model meets the needs of concurrent, distributed systems|actors-intro]]
+* [[Overview of Akka libraries and modules|modules]].
+* A [[more complex example|quickstart]] that builds on the Hello World example to illustrate common Akka patterns. 
```

Comment:
perhaps we should rename `quickstart.md` to something else to avoid confusion with http://developer.lightbend.com/guides/akka-quickstart-scala ?